### PR TITLE
doc: add `Mentions` to v5-note

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     add `Mentions` to v5-note      |
| 🇨🇳 Chinese |     添加`Mentions` 到文档     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: v5-note.md [hunk 1]
```diff
@@ -19,4 +19,5 @@
   - TreeSelect 组件
   - TimePicker 组件
   - DatePicker 组件
+  - Mentions 组件
```
Comment (afc163) [inline v5-note.md:22]: 为啥这几个要退一格

**Comment by MadCcc** [comment]
https://github.com/ant-design/ant-design/pull/37314
这个合了，之后都写到迁移文档里吧

File: v5-note.md (hunk 1 above)
Comment (heiyu4585) [reply v5-note.md:22]: > 为啥这几个要退一格

应该是啥样,没理解

Comment (afc163) [reply v5-note.md:22]: 图片

Comment (heiyu4585) [reply v5-note.md:22]: 上面有一级

**Comment by heiyu4585** [comment]
> #37314 这个合了，之后都写到迁移文档里吧

ok

**Pull request closed without merging**